p2p_wallet/SwiftUI/Spinner/Spinner.swift

```diff
@@ -1,18 +1,16 @@
-//
-//  Spinner.swift
-//  p2p_wallet
-//
-//  Created by Ivan on 04.10.2022.
-//
-
 import SwiftUI
 
 struct Spinner: View {
```

Comment:
well, I have create quite the same view and it called "NewCircularProgressIndicator", I guess we need to keep only one with a task @bigearsenal

Reply:
Great, I believe we can get rid of one of them

Reply:
@lisemyon lets remove one of them directly as part of striga implementation